### ISSUE TITLE
Kabsch Alignment

### ISSUE THREAD

**Issue by Leo-T-Zang** [NVIDIA-Digital-Bio/proteina#16]
Hi,

I noticed that in `proteinfoundation/proteinflow/proteina.py`, there's the following function:

```python
def align_wrapper(self, x_0, x_1, mask):
    """Performs Kabsch on the translation component of x_0 and x_1."""
    return kabsch_align(mobile=x_0, target=x_1, mask=mask)
```

However, I couldn't find any reference to this being used in the training loop. Is this a legacy function? I'm particularly curious whether, during training, the prediction and ground truth are ever aligned (e.g., using the Kabsch algorithm) for loss computation—especially after any rotation augmentations.

Thanks!

**Comment by Leo-T-Zang** [comment]
Thanks a lot!!
